accounts/views.py

```diff
@@ -0,0 +1,24 @@
+from django.shortcuts import render, redirect
+from django.contrib.auth.decorators import login_required
+
+from .forms import EditProfileForm
+
+@login_required
+def edit_profile(request):
+    """ 
+        If the request is POST and the form is valid, save the form and redirect to profile.
+        Otherwise, display current user instance in EditProfileForm on edit_profile.html.
+    """
+    if request.method == 'POST':
+        form = EditProfileForm(request.POST, instance=request.user)
+       
+        if form.is_valid():
```

Comment:
I think you need to add another `else` here for when `form.is_valid()` is not `True`.